packages/hydrogen-edge-adapter/README.md

```diff
@@ -0,0 +1,57 @@
+## Hydrogen Remix Adapter for Netlify
+
+The Hydrogen Remix Adapter for Netlify allows you to deploy your [Shopify Hydrogen](https://hydrogen.shopify.dev/) (via
+[Remix](https://remix.run)) app to [Netlify Edge Functions](https://docs.netlify.com/edge-functions/overview/).
+
+### Usage
+
+1. Modify your `server.js` file to use Netlify’s Hydrogen Edge Adapter:
+
+```js
+import { generateHandlerFunction, netlifyEdgeConfig } from '@netlify/hydrogen-edge-adapter'
+
+// Project-specific imports
+import { HydrogenSession } from '~/lib/session.server'
+import { getLocaleFromRequest } from '~/lib/utils'
+
+export default generateHandlerFunction({
+  HydrogenSession,
+  getLocaleFromRequest,
+})
+
+// Netlify Edge Function configuration
+export { netlifyEdgeConfig as config }
+```
+
+2. Modify your `remix.config.js` with Netlify configuration changes:
+
+```js
+const baseConfig = require('@netlify/hydrogen-edge-adapter/config')
+
+/** @type {import('@remix-run/dev').AppConfig} */
+module.exports = {
+  appDirectory: 'app',
+  // (some Remix configuration omitted for brevity)
+
+  // Netlify Overrides
+  ...baseConfig,
+}
+```
+
+3. Add a `netlify.toml` file using the Netlify Hydrogen Build plugin (this may be automated later):
```

Comment:
```suggestion
3. Add a `netlify.toml` file using the `@netlify/hydrogen-netlify-plugin` build plugin:
```

Was this a dev comment?

Reply:
Not exactly. It was a reference to the fourth bullet point on the PR readme:
> File an issue upstream regarding the hardcoded remix.config.js values for Hydrogen projects, which are the only reason the build plugin is needed. If we can get rid of these upstream, we can remove the Netlify build plugin. See https://github.com/Shopify/hydrogen/blob/180d7172fe1d16b3f1c34852f8adcdaf32d573ec/packages/cli/src/lib/config.ts#L113

But this detail can be left off the README, too!